<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="lenovo" failures="0" tests="1" name="Default test" time="0.083" errors="0" timestamp="2022-06-28T23:25:57 IST">
  <testcase classname="com.Azure.TestBase.AzureTestBase" name="@BeforeClass setup" time="0.008">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;java.lang.Class.getName()&amp;quot; because &amp;quot;pClass&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "java.lang.Class.getName()" because "pClass" is null
at org.testng.log4testng.Logger.getLevel(Logger.java:421)
at org.testng.log4testng.Logger.getLogger(Logger.java:150)
at com.Azure.TestBase.AzureTestBase.setup(AzureTestBase.java:30)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeClass setup -->
  <testcase classname="com.Azure.TestBase.AzureTestBase" name="@BeforeMethod start" time="-1.656438957843E9">
    <skipped/>
  </testcase> <!-- @BeforeMethod start -->
  <testcase classname="com.Azure.TestBase.AzureTestBase" name="@AfterMethod teardown" time="-1.656438957854E9">
    <skipped/>
  </testcase> <!-- @AfterMethod teardown -->
  <testcase classname="com.Azure.TestBase.AzureTestBase" name="@AfterClass end" time="-1.656438957857E9">
    <skipped/>
  </testcase> <!-- @AfterClass end -->
  <testcase classname="com.Azure.TestLayer.ValidLogin" name="testValidLgin" time="0.0">
    <skipped/>
  </testcase> <!-- testValidLgin -->
</testsuite> <!-- Default test -->
